requirements/deploy.txt

```diff
@@ -190,15 +190,15 @@ drf-extensions==0.7.1
     # via -r requirements/pip.txt
 drf-flex-fields==1.0.2
     # via -r requirements/pip.txt
-elastic-transport==8.13.1
+elastic-transport==8.15.0
     # via
     #   -r requirements/pip.txt
     #   elasticsearch
-elasticsearch==8.14.0
+elasticsearch==8.15.0
```

Comment:
I think the ES upgrade breaks one of our tests. cc @stsewd -- should we skip it for now?

Reply:
tests from ext-theme passed. I just re-triggered the task and not it's passing. I'm not sure if it was this upgrade or if this test was alrady flaky.